Docs/General/BasicGit.md

```diff
@@ -11,9 +11,14 @@ This means that while one person works on code for the autonomous, another could
 
 ### Resources
 
+Read one of the following:
```

Comment:
one or both?

Reply:
the resources are redundant w each other i think